biolink-model.yaml

```diff
@@ -5762,6 +5892,8 @@ classes:
     union_of:
       - disease
       - phenotypic feature
+    exact_mappings:
```

Comment:
As @kshefchek noted earlier, this is misleading. 

`HP:0000001` is the root of all HP terms (which includes terms besides phenotypic features).

I think a better substitute would be `HP:0000118`. @kshefchek would that be appropriate?

Reply:
HP:0000118 is a narrow mapping of phenotypic feature

Reply:
Ah, yes. Thanks for pointing that out 👍

Reply:
HP:0000118 is already mapped  to `phenotypic feature`. That said, another  PR https://github.com/biolink/biolink-model/pull/589  models  `pathological entity`  concepts  within which HP:0000118 may have a more exact mapping.